build.rs

```diff
@@ -3,7 +3,28 @@ use std::{env, path::PathBuf};
 
 fn main() {
     if cfg!(feature = "build-nng") {
-        let dst = cmake();
+        let generator = generator();
+        let stats = if cfg!(feature = "nng-stats") {
+            "ON"
+        } else {
+            "OFF"
+        };
+        let tls = if cfg!(feature = "nng-tls") {
+            "ON"
+        } else {
+            "OFF"
+        };
+
+        // Run cmake to build nng
+        let dst = Config::new("nng")
+            .generator(generator)
+            .define("CMAKE_BUILD_TYPE", "Release")
```

Comment:
The CMake crate will select the appropriate build type as long as we don't define it. See the docs [here][1].

[1]: https://docs.rs/cmake/0.1.35/cmake/struct.Config.html#method.profile

Reply:
ok